examples/step-18/step-18.cc

```diff
@@ -1333,6 +1288,13 @@ namespace Step18
           std::pair<double, std::string>(present_time, pvtu_master_filename));
         std::ofstream pvd_output("solution.pvd");
         DataOutBase::write_pvd_record(pvd_output, times_and_names);
+
+        std::ofstream visit_output("solution.visit");
+        static std::vector<std::pair<double, std::vector<std::string>>>
+          times_and_pieces;
```

Comment:
The use of a `static` variable is really not very pleasant here because it means that you can't call the simulator twice in a row, or twice in parallel. I think it would be nicer to put this into a member variable instead.

But maybe we don't need this -- does Visit now read .pvd files?